string-plugin/src/test/java/com/github/underscore/string/StringTest.java

```diff
@@ -1977,84 +1977,84 @@ public void chain() {
         $.chain(new Integer[] {0}).sort();
         $.chain(new Integer[] {0}).sortWith(new Comparator<Integer>() {
             public int compare(Integer value1, Integer value2) { return value1; } });
-        $.chain(new Integer[] {0}).sortBy(new Function1<Integer, Integer>() {
+        $.chain(new Integer[] {0}).sortBy(new Function<Integer, Integer>() {
             public Integer apply(Integer value) { return value; } });
         $.chain(new LinkedHashMap<Integer, Integer>().entrySet()).sortBy("");
-        $.chain(new Integer[] {0}).groupBy(new Function1<Integer, Integer>() {
+        $.chain(new Integer[] {0}).groupBy(new Function<Integer, Integer>() {
             public Integer apply(Integer value) { return value; } });
         $.chain(new Integer[] {0}).indexBy("");
-        $.chain(new Integer[] {0}).countBy(new Function1<Integer, Integer>() {
+        $.chain(new Integer[] {0}).countBy(new Function<Integer, Integer>() {
             public Integer apply(Integer value) { return value; } });
         $.chain(new Integer[] {0}).shuffle();
         $.chain(new Integer[] {0}).sample();
         $.chain(new Integer[] {0}).sample(1);
         $.chain(new int[] {0}).value();
-        $.chain(new String[] {""}).tap(new Block<String>() {
-            public void apply(String str) {
+        $.chain(new String[] {""}).tap(new Consumer<String>() {
+            public void accept(String str) {
             } });
-        $.chain(new String[] {""}).forEach(new Block<String>() {
-            public void apply(String str) {
+        $.chain(new String[] {""}).forEach(new Consumer<String>() {
+            public void accept(String str) {
             } });
-        $.chain(new String[] {""}).forEachRight(new Block<String>() {
-            public void apply(String str) {
+        $.chain(new String[] {""}).forEachRight(new Consumer<String>() {
+            public void accept(String str) {
             } });
         $.chain(new String[] {""}).every(new Predicate<String>() {
-            public Boolean apply(String str) { return true; } });
+            public boolean test(String str) { return true; } });
         $.chain(new String[] {""}).some(new Predicate<String>() {
-            public Boolean apply(String str) { return true; } });
+            public boolean test(String str) { return true; } });
         $.chain(new String[] {""}).contains("");
         $.chain(new String[] {""}).invoke("toString", Collections.emptyList());
         $.chain(new String[] {""}).invoke("toString");
         $.chain(new String[] {""}).pluck("toString");
         $.chain(new String[] {""}).where(Collections.<Tuple<String, String>>emptyList());
         $.chain(new String[] {""}).findWhere(Collections.<Tuple<String, String>>emptyList());
         $.chain(new Integer[] {0}).uniq();
-        $.chain(new Integer[] {0}).uniq(new Function1<Integer, Integer>() {
+        $.chain(new Integer[] {0}).uniq(new Function<Integer, Integer>() {
```

Comment:
\[FindBugs\] INFO: RV: Return value of $$Chain\.uniq\(Function\) ignored, is this OK in com\.github\.underscore\.string\.StringTest\.chain\(\)